docs/reference/commandline/node_promote.md

```diff
@@ -14,8 +14,10 @@ parent = "smn_cli"
 
     Promote a node as manager in the swarm
 
-Promotes a node that is pending a promotion to manager. This command targets a docker engine that is a manager in the swarm cluster.
+Promotes a node that is pending a promotion to manager.
```

Comment:
Not related to the change in this PR, but I noticed this is wrong. There is no such thing as pending a promotion to manager. It should say:

 "Promotes a worker node to a manager. Becoming a manager means the node will be able to accept administrative commands for the swarm, and become one of the candidates when the swarm elects a leader. The node will continue to be assigned tasks unless it is paused with `docker node pause` or drained with `docker node drain`. Only specific, stable nodes should be promoted to managers, because a majority of leaders must be responsive for the swarm to be available. Generally, a swarm should be set up with either 1, 3, or 5 managers."